filebeat/docs/modules/mysqlenterprise.asciidoc

```diff
@@ -0,0 +1,80 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-mysqlenterprise]]
+[role="xpack"]
+
+:modulename: mysqlenterprise
+:has-dashboards: false
+
+
+== MySQL Enterprise module
+beta[]
+
+This is a module for different types of MySQL logs. Currently focusing on data from the MySQL Enterprise Audit Plugin in JSON format.
+
+To configure the the Enterprise Audit Plugin to output in JSON format please follow the directions in the https://dev.mysql.com/doc/refman/8.0/en/audit-log-file-formats.html[MySQL Documentation]
+
+
+include::../include/gs-link.asciidoc[]
+
+
+[float]
+=== Compatibility
+
+This module has been tested against MySQL Enterprise 5.7.x and 8.0.x
+
+include::../include/configuring-intro.asciidoc[]
+
+:fileset_ex: audit
+
+include::../include/config-option-intro.asciidoc[]
+
+[float]
+==== `audit` fileset settings
+
+Example config:
+
+[source,yaml]
+----
+- module: mysqlenterprise
+  audit:
+    var.input: file
+    var.paths: /home/user/mysqlauditlogs/audit.*.log
+----
+
+include::../include/var-paths.asciidoc[]
+
+*`var.tags`*::
+
+A list of tags to include in events. Including `forwarded` indicates that the
+events did not originate on this host and causes `host.name` to not be added to
+events. Defaults to `[mysqlenterprise-audit]`.
+
+[float]
+==== MySQL Enterprise ECS Fields
+
+MySQL Enterprise Audit fields are mapped to ECS in the following way:
+[options="header"]
+|==============================================================
+| Check Point Fields         | ECS Fields                     |
```

Comment:
Is "Check Point" a concept in MySQL Audit logs or was this copied over from Check Point docs?